select/clear all added

sprajjwal opened the pull request:
added the option to select unselect all the options on the page.

ytsheng commented on src/App.js:
```diff
@@ -107,9 +109,24 @@ class App extends React.Component {
+    //     })
+    //   }
+    // } 
+  }
```
remove.

ytsheng commented on src/ExploreFunctions.js:
```diff
@@ -1,13 +1,31 @@
+//   }
+//   return true;
+// }
 
```
please remove all your commented out code, we don't want commented code in production

ytsheng commented on src/ExploreFunctions.js:
```diff
@@ -1,13 +1,31 @@
   render() {
     return (
         <div>
-            <p className="ExploreFunctions-header">Explore Functions:</p>
```
remove the spaces since the buttons are no longer added here.

ytsheng commented on src/App.js:
```diff
@@ -107,9 +109,24 @@ class App extends React.Component {
+    // } 
+  }
+
+  setAll(bool) {
```
rename the variable from bool to selected, bool in some languages are reserved.

sprajjwal commented:
Removed bogus code/comments. Updated the variable name!
the select all button should select all tags and clear all button will clear all tags.

![tnt](https://user-images.githubusercontent.com/19603200/83816827-2b3e2200-a678-11ea-94b4-551ac815558e.gif)

ytsheng commented:
Cool, looks good to me. Thank you so much!
